### PR TITLE
Add coverage of cython

### PR DESCRIPTION
**Pull request by lbdreyer**
Extends the code coverage testing to include cython files

### CODE REVIEW

**Comment by lbdreyer** [comment]
Not sure why this is failing :confused: it may be due to the coverage going down??

**Comment by lbdreyer** [comment]
This https://github.com/lemurheavy/coveralls-public/issues/565#issuecomment-120924694 suggests that we can set the tolerance to be something to allow for occasional decreases in code coverage. Unfortunately, I am not an owner so I cannot see the settings to change it.

@pelson @bjlittle I suspect one of you may an owner of this repo. Could you change the coverage tolerance setting https://coveralls.io/github/SciTools/cf-units?

File: setup.py [hunk 1]
```diff
@@ -55,7 +60,7 @@ def read(*parts):
                         **extra_extension_args)
 
 if cythonize:
-    [udunits_ext] = cythonize(udunits_ext)
+    [udunits_ext] = cythonize(udunits_ext, compiler_directives={'linetrace': True})
```
Comment (lbdreyer) [inline setup.py]: We could also add the `binding=True` directive?

Comment (lbdreyer) [inline setup.py:63]: We could also add the directive: `binding=True`. That can sometime help with cython code coverage.

**Comment by bjlittle** [comment]
@lbdreyer Closed by #126, thanks

**Pull request closed without merging**